readme.md

```diff
@@ -528,7 +528,7 @@ Please refer run with `--help` for latest information.
   Set to `True` to export the image information to a .XMP sidecar file, one per image in the album. The data contained within the file is the same but some software requires matching file names to detect the metadata. If set to `True`, then `writeImageXMP` is ignored.
 - verifyimage
 
-  Do image and zip checking after download. Set the value to `True` to enable.
+  Check if downloaded files is valid image or zip. Set the value to `True` to enable.
```

Comment:
If "files" is plural, the proper term to use would be "are", so `Check if downloaded files are a valid image or zip`, however this is not what `verifyimage` actually does. `verifyimage` simply checks if the file ends in ".jpg", ".png", ".gif", ".ugoira", or ".zip". It doe snot verify if it is a valid image or zip file.

Reply:
mistake.

that adding details is trying to make config PixivUtil2 easier for experienced users.

actually, `verifyimage` will validate files: https://github.com/Nandaka/PixivUtil2/blob/f2e9ab19fe471c93c0a1697d89f1f9f93f3c39cd/PixivDownloadHandler.py#L203-L248

Reply:
Oh so it does, my mistake

Reply:
it seems that the `overwrite` and `alwaysCheckFileSize` have too much tech debt.
and not sure how to detail those

Reply:
haha, it is from 2015 and no major rewrite at all. As long it is working as my expectation, I don't plan to do anything as long it doesn't break.